server/src/socobouser/routes/SocoboUserRoles.route.ts

```diff
@@ -0,0 +1,36 @@
+import { Router } from "express";
+import { ModelValidationHandler, ValidationGroup } from "../../app/index";
+import { AuthValidationHandler } from "../../auth/index";
+import { SocoboUserRolesHandler } from "../handlers/SocoboUserRoles.handler";
+import { SocoboUserRole } from "../index";
+
+export class SocoboUserRolesRoute {
+
+  constructor (
+    private _router: Router,
+    private _socoboUserRolesHandler: SocoboUserRolesHandler,
+    private _authValidationHandler: AuthValidationHandler,
+    private _modelValidationHandler: ModelValidationHandler
+  ) {}
+
+  public createRoutes (): Router {
+    this._router.get("/",
+      this._authValidationHandler.checkToken,
+      this._socoboUserRolesHandler.getAll);
+
+    this._router.get("/:id",
+      this._authValidationHandler.checkToken,
+      this._socoboUserRolesHandler.getById);
+
+    this._router.put("/:id",
```

Comment:
Delete and Update only for admins. Same as providers -> I'm not sure why we would like to delete roles